tplinkrouterc6u/client.py

```diff
@@ -132,10 +132,17 @@ async def _get_firmware(self) -> Firmware:
         return firmware
 
     async def _get_status(self) -> Status:
+
+        def _calc_cpu_usage(data: dict) -> float:
+            return (data['cpu_usage'] + data['cpu1_usage'] + data['cpu2_usage'] + data['cpu3_usage']) / 4
+
         data = await self._get_data('admin/status?form=all')
         status = Status
         status.devices = []
         status.macaddr = data['lan_macaddr']
+        status.wan_ipv4_uptime = data['wan_ipv4_uptime']
```

Comment:
I dont see parameter `wan_ipv4_uptime` in my router response. Im afraid not all routers it pass